packer/README.md

```diff
@@ -54,23 +58,33 @@ To test the newly created box:
 3.  vagrant up centos7-katello-3.18-stable
 4.  Verify that 'rpm -q katello-repos' is correct and the foreman and katello git repos are on the correct branches
 
-To upload the box:
-1. While you can try with the latest version of vagrant, versions 2.2.11-2.2.14 cannot properly upload boxes.  Download 2.2.9 (this will not interfere with your rpm installed vagrant):
-```
-wget https://releases.hashicorp.com/vagrant/2.2.9/vagrant_2.2.9_linux_amd64.zip
-unzip vagrant_2.2.9_linux_amd64.zip
-sudo mv vagrant /usr/bin/vagrant-upstream
+### Uploading a versioned box to Vagrant Cloud
+
+The version of Vagrant shipped with Fedora does not support Vagrant Cloud uploads (due to a library licensing issue):
+
+1. Deploy a Centos 7 VM (vagrant up centos7), install vagrant from
```

Comment:
Looks like you say `install vagrant from` twice in steps 1 and 2

Reply:
fixed and squashed!